util/test/sub_test

```diff
@@ -1311,6 +1318,14 @@ for testname in testsrc:
         print('[Elapsed compilation time for "{0}" - {1:.3f} '
             'seconds]'.format(test_name, elapsedCompTime))
 
+        # remove some_file: output from C compilers
+        if is_c_test:
+          for arg in args:
+            if arg.endswith(".c"):
+              # remove lines like
+              # somefile.c:
+              # that some C compilers emit when compiling multiple files
+              output = output.replace(arg + ":\n", "");
```

Comment:
I didn't see a way to stifle them. Both Cray and PGI compilers do it, but they do it in exactly the same format. This bit of code is also unlikely to affect errors, since it looks for filename:<newline> .